docs/core/preview3/tools/dependencies.md

```diff
@@ -27,12 +29,12 @@ The `<PackageReference>` has the following basic structure:
 </PackageReference>
 ```
 
-If you are familiar with MSBuild, it will look familiar to the other [reference types]() that already exist. The key is the `Include` statement which specifies the package id that you wish to add to the project. The `<Version>` child element specifies the version to get. The versions are specified as per [NuGet version rules](https://docs.nuget.org/ndocs/create-packages/dependency-versions#version-ranges).  
+If you are familiar with MSBuild, it will look familiar to the other reference types that already exist. The key is the `Include` statement which specifies the package id that you wish to add to the project. The `<Version>` child element specifies the version to get. The versions are specified as per [NuGet version rules](https://docs.microsoft.com/nuget/create-packages/dependency-versions#version-ranges).
 
 > [!NOTE]
-> If you are not familiar with the overall `csproj` syntax, you can use the [MSBuild project reference documentation]() to get acquainted.  
+> If you are not familiar with the overall `csproj` syntax, you can use the MSBuild project reference documentation to get acquainted.  
```

Comment:
Can we link to the MSBuild project reference documentation? The [project file schema](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-project-file-schema-reference) is available on docs.